client/src/controllers/KeyboardController.ts

```diff
@@ -49,6 +55,7 @@ export class KeyboardController extends Controller<
   static values = {
     key: { default: '', type: String },
     scope: { default: '', type: String },
+    action: { default: KeyboardAction.CLICK, type: String },
```

Comment:
Let's move the `action` to the top of the values, keep things alpha sorted unless there's a reason why, this helps keep the code maintainable and understood.

Same with the `actionValue`.

I can do this when merging though.

Also, let's default to an empty string, it's simpler and the code will work as is.